lib/services/coins/epiccash/epiccash_wallet.dart

```diff
@@ -1105,30 +1105,26 @@ class EpicCashWallet extends CoinServiceAPI {
         await testEpicboxServer(_epicBox.host, _epicBox.port as int);
 
     if (!connected) {
-      print("DEFAULT EPIC BOX IS NOT CONNECTED");
-
-      //Get all available hosts
-      final allBoxes = DefaultEpicBoxes.all;
-      final List<EpicBoxModel> alternativeServers = [];
+      //Default Epic Box is not connected, iterate through list of defaults
+      Logging.instance.log("Default Epic Box server not connected",
+          level: LogLevel.Warning);
 
-      for (var i = 0; i < allBoxes.length; i++) {
-        if (allBoxes[i].name != _epicBox?.name) {
-          alternativeServers.add(allBoxes[i]);
-        }
-      }
+      // Copy list of all default EB servers but remove currently-connected one
+      List<EpicBoxModel> alternativeServers = DefaultEpicBoxes.all;
+      alternativeServers.removeWhere((opt) => opt.name == _epicBox?.name);
```

Comment:
this needed to be `alternativeServers = alternativeServers.removeWhere((opt) => opt.name == _epicBox?.name);`

Reply:
No it didn't